R/read_dat_Witec.R

```diff
@@ -58,7 +58,7 @@ hySpc.testthat::test(read_dat_Witec) <- function() {
   test_that("-y file guessing", {
     spc <- read_dat_Witec(paste0(tmpdir, "/Witec-timeseries-x.dat"))
     spc$filename <- gsub("^.*/", "", spc$filename)
-
+    capture_output(paste0(tmpdir, "/Witec-timeseries-x.dat"))
```

Comment:
Why do we need to capture the output of `paste0()`? Why is the captured output not used?